Library/Homebrew/cmd/audit.rb

```diff
@@ -552,7 +552,7 @@ def audit_line(line, lineno)
     end
 
     if @strict
-      if line =~ /system (["'][^"' ]*\s[^"' ]*["'])/
+      if line =~ /system (["'][^"' ]*(?:\s[^"' ]*)+["'])/
```

Comment:
`(…)` is a *capturing group*, which you can later reference using `$1` (the first matching one), `$2` (the second), etc. `(?:…)` is a *non-capturing group*, meaning we don’t care about having it in the resulting match. See [this explanation on SO](http://stackoverflow.com/a/3513858/735926).

```irb
>> $1 if "bar foo qux" =~ /foo/
nil
>> $1 if "bar foo qux" =~ /(foo)/
"foo"
>> $1 if "bar foo qux" =~ /(?:foo)/
nil
```

This is used when you want to group stuff together but don’t want to capture it.